src/config/config.c

```diff
@@ -113,7 +104,7 @@ cfgCommandName(void)
     FUNCTION_TEST_VOID();
 
     ASSERT(configLocal != NULL);
-    ASSERT(configLocal->command < cfgCmdNone);
+    ASSERT(configLocal->command < CFG_COMMAND_TOTAL);
```

Comment:
The previous `Assert()` on :65 had been:

```diff
-    ASSERT(configLocal->command <= cfgCmdNone);
+    ASSERT(configLocal->command < CFG_COMMAND_TOTAL);
```
Here, we're changing `< cfgCmdNone` to the same value.  I would assume that the difference here is that the argument here could never have been `cfgCmdNone`, so practically-speaking these are equivalent, but just wanted to call this out in case this `<` vs `<=` is significant.

Reply:
The < vs <= difference is down to using a total vs the max enum. We could keep the same logic with `<= cfgCmdVerify` but then we'd need to worry about keeping that updated since there is no longer a deterministic last command. So we used the total and < instead.